internal/controller/api/v1alpha1/image_types.go

```diff
@@ -17,25 +17,28 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // ImageSpec defines the desired state of Image
 type ImageSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of Image. Edit Image_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// The name of the Cluster CR
+	ClusterName  *corev1.ObjectReference `json:"cluster_name,omitempty"`
```

Comment:
Nice is that used for secrets as well?

Reply:
Isn't it required? it should be without `omitempty`

Reply:
by the way i see that there is `*corev1.SecretReference` as well

Reply:
Done.
removed `omitempty` and using `corev1.SecretReference`
I'm not sure if it should be used as a pointer or not. seem to have an impact on `internal/controller/api/v1alpha1/zz_generated.deepcopy.go` but no visible change in CRD yaml.

Reply:
Yes, we should use the standard references that include namespace

Reply:
Done